python/lsst/daf/butler/registry/_dbAuth.py

```diff
@@ -131,6 +131,10 @@ def getAuth(self, drivername, username, host, port, database):
         the database connection URL, the dictionary must also contain a
         ``username`` item.
 
+        The ``url`` item of the authorization dictionaries can contain both the
+        dialect and the driver of the database URL but the driver is ignored
```

Comment:
Isn't that even more confusing?  Better to forbid it, I think.

Reply:
I was unsure of how opinionated we should make DbAuth. I raise an error when this happens and I have updated the docstrings to reflect that.